src/components/puck/Promo.tsx

```diff
@@ -0,0 +1,340 @@
+import * as React from "react";
+import { ComponentConfig, Fields } from "@measured/puck";
+import { Image, ImageProps, ImageType } from "@yext/pages-components";
+import {
+  yextCn,
+  useDocument,
+  resolveYextEntityField,
+  EntityField,
+  YextEntityField,
+  YextEntityFieldSelector,
+  NumberOrDefault,
+  HeadingProps,
+  NumberFieldWithDefaultOption,
+  CTAProps,
+  ImageWrapperProps,
+  Section,
```

Comment:
Move the component imports to the more specific imports like "./atoms/Body.js so that when it is pulled via the CLI it uses the local version. (Keep things like YextEntityField imported from index.js so that it gets imported from @yext/visual-editor when pulled down)